awswrangler/distributed/_distributed.py

```diff
@@ -119,6 +118,12 @@ def initialize_ray(
         Number of GPUs to assign to each raylet, by default 0
     """
     if not ray.is_initialized():
+        # Detect an existing cluster
+        ray_address = os.environ.get("RAY_ADDRESS")
```

Comment:
So `RAY_ADDRESS` gets passed as an environment variable when a cluster is already running on your machine?

Reply:
Yes, this is on of the ways Ray detects an existing cluster if the address is not provided. I "lifted" this from ray code as we need to clearly distinguish between these two cases - whether we are connecting to an existing cluster or starting a new local one because ray will throw exceptions if you try to configure cluster parameters when existing cluster is running.